### PR TITLE
fix: chainsetup

### PR DESCRIPTION
**Pull request by weichweich**
## How to test:

Create a local setup with multiple nodes and custom keys. (follow the "start a private network" tutorial on substrate.dev)

1. [generate keys](https://substrate.dev/docs/en/tutorials/start-a-private-network/keygen)
1. [create chain spec](https://substrate.dev/docs/en/tutorials/start-a-private-network/customspec)
2. [start chain with keys](https://substrate.dev/docs/en/tutorials/start-a-private-network/customchain)

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

**Comment by weichweich** [comment]
troubleshoot:

1. make sure to use ed25519 public key for grandpa and sr25519 for aura (but same mnemonic)
2. block production will only start if we are connected to a second node. Therefore start a second node and connect both.
3. each node gets their own private key (via curl)
4. verify that the authorities (query aura) on chain are the same as the uploaded keys in the keystore.

File: src/chain_spec.rs [hunk 1]
```diff
@@ -129,20 +131,16 @@ impl Alternative {
...
-					// Testnet Faucet accounts
-					hex!("3ba6e1019a22234a9349eb1d76e02f74fecff31da60a0c8fc1e74a4a3a32b925").into(),
-					hex!("b7f202703a34a034571696f51e95047417956337c596c889bd4d3c1e162310b6").into(),
-					hex!("5895c421d0fde063e0758610896453aec306f09081cb2caed9649865728e670a").into()
-				],
-							true,
+								// Testnet Faucet accounts
+								TEST_AUTH_ALICE.into(),
+								TEST_AUTH_BOB.into(),
+							],
```
Comment (Dudleyneedham) [inline src/chain_spec.rs:143]: Add money for CHARLIE!

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -26,7 +26,7 @@ version = '1.2.1'
 default-features = false
 features = [ 'derive' ]
 package = 'parity-scale-codec'
-version = '1.3.0'
+version = '1.3.1'
```
Comment (Dudleyneedham) [inline runtime/Cargo.toml:29]: Great!

File: src/chain_spec.rs [hunk 3]
```diff
@@ -129,20 +131,16 @@ impl Alternative {
...
 							vec![
-					// Testnet Faucet accounts
-					hex!("3ba6e1019a22234a9349eb1d76e02f74fecff31da60a0c8fc1e74a4a3a32b925").into(),
-					hex!("b7f202703a34a034571696f51e95047417956337c596c889bd4d3c1e162310b6").into(),
-					hex!("5895c421d0fde063e0758610896453aec306f09081cb2caed9649865728e670a").into()
-				],
-							true,
+								// Testnet Faucet accounts
+								TEST_AUTH_ALICE.into(),
+								TEST_AUTH_BOB.into(),
```
Comment (Dudleyneedham) [inline src/chain_spec.rs:142]: Shouldn't charlie have money?

File: start-node.sh [hunk 4]
```diff
@@ -121,7 +125,10 @@ fi
 
 # NODE_SEED = The seed for the validator account to be used. Has to be combined with NODE_KEY
 if [[ "$validator" = "1" ]]; then
```
Comment (Dudleyneedham) [inline start-node.sh:127]: As you told me in person.

File: runtime/Cargo.toml (hunk 2 above)
Comment (weichweich) [reply runtime/Cargo.toml:29]: THX

File: src/chain_spec.rs (hunk 3 above)
Comment (weichweich) [reply src/chain_spec.rs:142]: DOOOOONE

File: start-node.sh (hunk 4 above)
Comment (weichweich) [reply start-node.sh:127]: Solved!

**Pull request merged**